.github/workflows/build-notebooks-TEMPLATE.yaml

```diff
@@ -143,28 +183,13 @@ jobs:
                 image \
                 --image-src podman \
                 --podman-host /var/run/podman/podman.sock \
-                --scanners vuln,secret \
+                --scanners vuln,secret --ignore-unfixed \
```

Comment:
Have we already discussed not running the `secret` scanner for PRs? I think not yet. That's the main part that's taking up time, as trivy itself helpfully points out in logs.

Reply:
I almost removed `secret` on this PR because yeah it takes more time and I haven't seen any issue related to that. But then thought that we could miss issues if we remove it. Anyway, I'm +1 for removing it.